frontend-react/src/content/resources/reportstream-api/documentation/SamplePayloadsAndOutput.mdx

```diff
@@ -49,16 +49,16 @@ Response
 
 Input
 
-<a href="https://drive.google.com/file/d/1v60xjAKSyQ4g0toxCC82AGkhhWxwCnn5/view?usp=share_link" className="usa-button">Download a sample file <Icon.FileDownload /></a>
+<a href="https://drive.google.com/file/d/1v60xjAKSyQ4g0toxCC82AGkhhWxwCnn5/view" className="usa-button">Download a sample file <Icon.FileDownload /></a>
```

Comment:
File needs to be downloaded locally, placed in the project's public/assets folder and linked to here via /assets